core/example/solvers/direct.cpp

```diff
@@ -0,0 +1,95 @@
+#include "definitions.h"
+#include "direct.h"
+#include <Cabana_AoSoA.hpp>
+#include <Cabana_Parallel.hpp>
+#include <Cabana_ExecutionPolicy.hpp>
+#include <Cabana_MemberTypes.hpp>
+
+TDS::TDS(int periodic)
+{
+  // number of periodic images (number of shells surrounding systems)
+  _periodic_shells = periodic;
+}
+
+TDS::~TDS()
+{
+}
```

Comment:
There is no point in providing that destructor.  Get rid of it.

Reply:
Removed unneeded destructors.